### PR TITLE
add hyperlane docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

This PR adds translations for the Hyperlane docs
https://github.com/PureStake/moonbeam-docs/pull/481/files

LayerZero isn't completed yet, so I'll create a separate PR for that.

I checked this using `BuildDocsSite` and all looks good 🚀

### CODE REVIEW

**Review by albertov19 — APPROVED** [review]
LGTM, but please wait until Xinxin reviews this before merging ;)

File: builders/interoperability/protocols/hyperlane.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# Hyperlane协议
+
+![Hyperlane Moonbeam banner](/images/builders/integrations/bridges/hyperlane/hyperlane-banner.png)
+
+## 概览 {: #introduction }
+
+[Hyperlane](https://hyperlane.xyz){target=_blank}是Web3的一个安全模块化的跨链通信协议，能够使dApp用户与任何资产或应用在任何互连的链上一键式交互。该协议支持通用资产传输以及自定义跨链通信。
+
+Hyperlane使用一个称为[主权共识](https://docs.hyperlane.xyz/hyperlane-docs-1/protocol/security/sovereign-consensus){target=_blank}的方式，允许开发者通过发送的消息和已验证的跨链配置该方式。Hyperlane由验证人、中继器和watchtowers组成。验证人将监视并确认跨链消息。中继器支出gas，跨链发送消息。Watchtowers检查确保验证人是善意的参与者，从而保证协议的安全性。如需了解更多，请查看技术堆栈图及其[协议文档](https://docs.hyperlane.xyz/hyperlane-docs-1/protocol/overview){target=_blank}。
```
Comment (wuxinxin2000) [inline builders/interoperability/protocols/hyperlane.md:14]:
```suggestion
Hyperlane使用一个称为[主权共识](https://docs.hyperlane.xyz/hyperlane-docs-1/protocol/security/sovereign-consensus){target=_blank}的方式，允许开发者通过配置该方式来跨链发送消息和验证消息。Hyperlane由验证人、中继器和瞭望塔（watchtowers）组成。验证人将监视并确认跨链消息。中继器支出gas，跨链发送消息。瞭望塔检查确保验证人是善意的参与者，从而保证协议的安全性。如需了解更多，请查看技术栈图及其他们的[协议文档](https://docs.hyperlane.xyz/hyperlane-docs-1/protocol/overview){target=_blank}。
```

File: builders/interoperability/protocols/hyperlane.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+![Hyperlane Technology Stack diagram](/images/builders/integrations/bridges/hyperlane/hyperlane-1.png)
+
+Hyperlane API为开发Web3应用提供了丰富的套件，确保开发者拥有构建所需的工具。借助这些工具和API，开发者可以使用Hyperlane协议及其API，来编写可以轻松部署在与所有Hyperlane连接的生态系统dApp。
```
Comment (wuxinxin2000) [inline builders/interoperability/protocols/hyperlane.md:18]:
```suggestion
Hyperlane API为开发Web3应用提供了丰富的套件，确保开发者拥有构建所需的工具。借助这些工具和API，开发者可以使用Hyperlane协议及其API，来编写可以轻松部署在与所有Hyperlane连接的生态系统的dApp。
```

File: builders/interoperability/protocols/hyperlane.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+--8<-- 'text/disclaimers/third-party-content-intro.md'
+
+## 开始操作 {: #getting-started }
+
+开始使用Hyperlane构建跨链应用程序可能所需资源：
```
Comment (wuxinxin2000) [inline builders/interoperability/protocols/hyperlane.md:24]:
```suggestion
开始使用Hyperlane构建跨链应用程序可能所需的资源：
```

**Review by wuxinxin2000 — APPROVED** [review]

File: builders/interoperability/protocols/hyperlane.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+title: 通过Hyperlane进行跨链操作
+description: 学习如何使用Hyperlane协议和API桥接资产以及将Moonbeam dApp连接到多个区块链上的资产和功能的其他方法。
+---
+
+# Hyperlane协议
+
+![Hyperlane Moonbeam banner](/images/builders/integrations/bridges/hyperlane/hyperlane-banner.png)
```
Comment (eshaben) [inline builders/interoperability/protocols/hyperlane.md:8]:
```suggestion
![Hyperlane Moonbeam banner](/images/builders/interoperability/protocols/hyperlane/hyperlane-banner.png)
```

File: builders/interoperability/protocols/hyperlane.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+## 概览 {: #introduction }
+
+[Hyperlane](https://hyperlane.xyz){target=_blank}是Web3的一个安全模块化的跨链通信协议，能够使dApp用户与任何资产或应用在任何互连的链上一键式交互。该协议支持通用资产传输以及自定义跨链通信。
+
+Hyperlane使用一个称为[主权共识](https://docs.hyperlane.xyz/hyperlane-docs-1/protocol/security/sovereign-consensus){target=_blank}的方式，允许开发者通过配置该方式来跨链发送消息和验证消息。Hyperlane由验证人、中继器和瞭望塔（watchtowers）组成。验证人将监视并确认跨链消息。中继器支出gas，跨链发送消息。瞭望塔检查确保验证人是善意的参与者，从而保证协议的安全性。如需了解更多，请查看技术栈图及其他们的[协议文档](https://docs.hyperlane.xyz/hyperlane-docs-1/protocol/overview){target=_blank}。
+
+![Hyperlane Technology Stack diagram](/images/builders/integrations/bridges/hyperlane/hyperlane-1.png)
```
Comment (eshaben) [inline builders/interoperability/protocols/hyperlane.md:16]:
```suggestion
![Hyperlane Technology Stack diagram](/images/builders/interoperability/protocols/hyperlane/hyperlane-1.png)
```

**Pull request merged**